cidc_api/config/settings.py

```diff
@@ -68,6 +68,5 @@
 ITEM_METHODS = ["GET", "PATCH"]
 CACHE_CONTROL = "no-cache"
 DOMAIN = get_DOMAIN()
-PAGINATION_DEFAULT = 100
-PAGINATION_LIMIT = 200
```

Comment:
wouldn't removing the limit setting make the limit to be equal to the limit default which is 50 acc to docs?

Reply:
Missed the detail that `PAGINATION_LIMIT` defaults to 50.